jdat_notebooks/MIRI_IFU_YSOs_in_the_LMC/isha_nayak_ysos_in_the_lmc.ipynb

```diff
@@ -4,14 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# JWST Data Analysis"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "# Young Stellar Objects in the Large Magellanic Cloud"
+    "# IFU of YSOs in LMC"
```

Comment:
This change makes it less readable because you replaced some descriptive words with a bunch of acronyms. Is this intentional?

Reply:
There are also no categories in this notebook. Is that intentional?